modules/exoplanets/tasks/exoplanetsdatapreparationtask.cpp

```diff
@@ -88,25 +88,36 @@ void ExoplanetsDataPreparationTask::perform(
     int version = 1;
     binFile.write(reinterpret_cast<char*>(&version), sizeof(int));
 
-    std::string planetRow;
-    getline(inputDataFile, planetRow); // The first line, containing the data names
+    auto readFirstDataRow = [](std::ifstream& file, std::string& line) -> void {
+        while (getline(file, line)) {
+            bool shouldSkip = line[0] == '#' || line.empty();
+            if (!shouldSkip) break;
+        }
+    };
+
+    // Find the line containing the data names
+    std::string columnNamesRow;
+    readFirstDataRow(inputDataFile, columnNamesRow);
 
     // Read column names into a vector, for later access
     std::vector<std::string> columnNames;
-    std::stringstream sStream(planetRow);
+    std::stringstream sStream(columnNamesRow);
     std::string colName;
     while (getline(sStream, colName, ',')) {
         columnNames.push_back(colName);
     }
 
     // Read total number of items
     int total = 0;
-    while (getline(inputDataFile, planetRow)) {
+    std::string row;
+    while (getline(inputDataFile, row)) {
```

Comment:
`std::getline`  otherwise you're relying on the namespace deduction based on the input argument to detect the right `getline` function, which seems unnecessary in this case